packages/react/src/components/FileUploader/FileUploader-test.js

```diff
@@ -255,7 +255,13 @@ describe('FileUploader', () => {
 });
 
 describe('FileUploaderDropContainer', () => {
-  const dropContainer = <FileUploaderDropContainer className="extra-class" />;
+  const onAddFiles = jest.fn();
+  const dropContainer = (
```

Comment:
We should move blocks where we create components into `beforeEach` or a similar generator function to prevent test flakiness

Reply:
@joshblack Agreed on `beforAll()`/`beforeEach()` making tests more stable, but it's orthogonal to this PR. Do you want to enter an issue for that?